### PR TITLE
adding anmolpant.md profile page

### PR DESCRIPTION
**Pull request by anmolpant**
Please add me to the organization. 
Thank you.

### CODE REVIEW

File: members/anmolpant.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+             |     ,','      |      |    | |
+             |   ,','    ____|_____/    /  |
+             | ,','  __/ |             /   |
+_____________|','   ///_/-------------/   |
+              |===========,'
+```
+
+<div align="center">
+<img src="https://i.pinimg.com/originals/a4/f2/cb/a4f2cb80ff2ae2772e80bf30e9d78d4c.gif" width="300" height="300" />
+</div>
```
Comment (RishabhKodes) [inline members/anmolpant.md:39]: You can remove the <div> tags for the GIF to work properly.

Comment (xlogix) [reply members/anmolpant.md:39]: Done with it?

Comment (anmolpant) [reply members/anmolpant.md]: Yes, I have changed my readme.md file and have incorporated the above changes in my next pr.

**Comment by xlogix** [comment]
Great! Welcome to the community!

**Pull request merged**